packages/nvidia_nat_ragas/src/nat/plugins/ragas/rag_evaluator/evaluate.py

```
@@ -114,67 +60,47 @@ def extract_input_obj(self, item: EvalInputItem) -> str:
+        retrieved_contexts = intermediate_step_adapter.get_context(item.trajectory, event_filter)
+
+        return SingleTurnSample(user_input=user_input,
+                                reference=reference,
+                                response=response,
+                                reference_contexts=reference_contexts,
+                                retrieved_contexts=retrieved_contexts)
+
+    async def evaluate_item(self, item: EvalInputItem) -> EvalOutputItem:
+        """Run configured ragas metric for one eval item and return one output item."""
+        if not self.metrics:
+            raise ValueError("No RAGAS metrics configured.")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)